script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -367,87 +381,90 @@ The unidentified monster holds up its numerous disgusting arms toward Ciel, who
 [sha:14920a5a534b6f0efc72d6439a132291d68494b8]{
 -- Page 283, Offset 32449.
 -- 鞭のように伸縮する腕。
-The arms are released like a whip.
+The arms lash out like whips.
 }
 [sha:9d84c9d2afd8681010c822d13cd5499a015cb264]{
 -- Page 283, Offset 32450.
 -- その一本一本から死徒と同じ匂いを嗅ぎ取って、シエルの思考はより冷静になっていく。
-Recognizing the stench of each arm to be identical to a Dead Apostle, Ciel manages to regain her composure.
+Recognizing the stench they give off as that of a Dead Apostle, Ciel manages to regain her composure.
 }
 [sha:bd72dc27e4af04c6aa66295531d6f8e71f0a1ceb]{
 -- Page 284, Offset 32451.
 -- 「新種の<屍鬼|グール>、なんですね……！」
-"So this must be a new species of Ghoul...!"
+"So this is a new species of Ghoul...!"
 }
 [sha:b7f32a81b310296f629f7c1c8ece4959c64a4064]{
 -- Page 284, Offset 32452.
 -- ロアが配置した対<自分|シエル>用の防衛措置。
-A defensive measure that Roa deployed against her.
+Roa must have stationed it here to stand guard against her.
 }
 [sha:456eede67187ef83e71b727bc4649c5077db7f8d]{
 -- Page 284, Offset 32453.
 -- そう考えれば驚く事は何もない。
-In that sense, there's nothing to be surprised about.
+It makes complete sense when you think about it.
 }
 [sha:b4a18605ed6344e671f8b42945d9792a13d646d1]{
 -- Page 284, Offset 32454.
 -- シエルは地上から、壁に張り付いた大型屍鬼の生態を観察する。
-From above ground, Ciel observes the enormous Ghoul clinging to the walls.
+Still standing on the ground, Ciel stakes stock of the enormous Ghoul clinging to the wall.
 }
 [sha:168b5422df0cb4e0f96cfff7f1cb77a6f5791154]{
 -- Page 285, Offset 32455.
 -- 全身凶器の塊だが、注目すべきは二点だけ。
-Although its entire body is a cluster of lethal weapons, there are only two crucial points to observe.
+Although its entire body bristles with lethal weapons,
+ there are only two that deserve special attention.
 }
 [sha:e14e5456f2f1d73f7a89edb95604ce3222316a4f]{
 -- Page 285, Offset 32456.
 -- 肩の瘤から群生した牙と、巨大な左腕。
-The tusks that sprout from the lump on its shoulder, and its enormous left arm.
+The tusks sprouting from the lump on its shoulder, and its enormous left arm.
```

Comment:
i havent looked in game but since its a spider-y thing, is this "fangs"?

Reply:
I'd say tusk is more accurate, they're massive